### PR TITLE
List User / Add User (without connection to backend)

### PR DESCRIPTION
**Pull request by e-i-n-s**
Please review

### CODE REVIEW

File: app/view_users/panel_editUser_test.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            var formElement = angular.element(templateHtml);
+            $compile(formElement)(scope);
+            scope.$apply()
+        }));
+
+        it('should be Defined', inject(function () {
+            expect(panelDialogCtrl).toBeDefined();
+        }));
+
+        it('should set title of panel to "Edit User"', inject(function () {
```
Comment (mawit) [inline app/view_users/panel_editUser_test.js]: Shouldn't this be "Add a new User" instead of "Edit User"?

File: app/view_users/view_users.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            <div flex="30">Name</div>
+            <div flex="20">Phone Number</div>
+            <div flex layout="row" layout-align="end">Operations</div>
+        </div>
+
+    </md-subheader>
+
+    <md-divider></md-divider>
+
+    <md-list-item flex ng-repeat="sipUser in sipUsers">
```
Comment (mawit) [inline app/view_users/view_users.html]: May be we can add a filter here to bo able to search for a specified user

File: app/view_users/view_users.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                        return true;
+                    }
+                });
+                if (!found) {
+                    $scope.sipUsers.push(user);
+                }
+            }
+        };
+
+        this.showDialog = function ($user) {
```
Comment (mawit) [inline app/view_users/view_users.js:36]: We will have to use this alot, I think this should be global.

File: e2e-tests/scenarios.js [hunk 4]
```diff
@@ -2,41 +2,130 @@
...
+            expect(element.all(by.css('[ng-view] p')).first().getText()).toMatch(/version of the dashboard/);
+        });
 
-    it('should render dashboard when user navigates to /dashboard', function() {
-      expect(element.all(by.css('[ng-view] p')).first().getText()).
-        toMatch(/version of the dashboard/);
     });
 
-  });
+    describe('view2', function () {
```
Comment (mawit) [inline e2e-tests/scenarios.js]: Why is this inside the dashboard block?

File: e2e-tests/scenarios.js [hunk 5]
```diff
@@ -2,41 +2,130 @@
...
+    });
 
 
-  describe('dashboard', function() {
+    describe('dashboard', function () {
 
-    beforeEach(function() {
-      browser.get('index.html#!/dashboard');
-    });
+        beforeEach(function () {
```
Comment (mawit) [inline e2e-tests/scenarios.js]: Wrong indention?

File: e2e-tests/scenarios.js [hunk 6]
```diff
@@ -2,41 +2,130 @@
```
Comment (mawit) [inline e2e-tests/scenarios.js]: Could this file be splitted into multipple?

**Comment by mawit** [comment]
I wrote my comments directly in to the changes. I thing we should discuss this tomorrow.

File: app/view_users/panel_editUser_test.js (hunk 1 above)
Comment (e-i-n-s) [reply app/view_users/panel_editUser_test.js]: Yes, I will correct it

Comment (RafaelOstertag) [reply app/view_users/panel_editUser_test.js]: @e-i-n-s I thought you're only SM? Are involved in the frontend, too?

File: e2e-tests/scenarios.js (hunk 6 above)
Comment (e-i-n-s) [reply e2e-tests/scenarios.js]: I prefer...

File: e2e-tests/scenarios.js (hunk 5 above)
Comment (e-i-n-s) [reply e2e-tests/scenarios.js]: Should be ok

File: e2e-tests/scenarios.js (hunk 4 above)
Comment (e-i-n-s) [reply e2e-tests/scenarios.js]: irrelevant when we split the E2E test files

File: app/view_users/view_users.js (hunk 3 above)
Comment (e-i-n-s) [reply app/view_users/view_users.js:36]: to discuss

File: app/view_users/view_users.html (hunk 2 above)
Comment (e-i-n-s) [reply app/view_users/view_users.html]: I will implement this

**Comment by e-i-n-s** [comment]
Should we rename "view2" to "view_template" or remove it?

**Pull request merged**